plugin.rb

```diff
@@ -310,15 +310,16 @@ def self.skip_db?
   end
 
   if respond_to?(:register_modifier)
-    register_modifier(:search_rank_sort_priorities) do |priorities, search|
+    register_modifier(:search_rank_sort_priorities) do |priorities, _search|
       if SiteSetting.prioritize_solved_topics_in_search
         condition = <<~SQL
-          EXISTS
-            (
-              SELECT 1 FROM topic_custom_fields f
-              WHERE topics.id = f.topic_id
-              AND f.name = '#{::DiscourseSolved::ACCEPTED_ANSWER_POST_ID_CUSTOM_FIELD}'
-            )
+          EXISTS (
```

Comment:
Formatted the query and added the "`AND value IS NOT NULL`" condition that is being used everywhere else